### PR TITLE
LMBQ-373: Fix visual verification tests instead of using Firefox and skipping them

### PR DESCRIPTION
**Pull request by DemeSzabolcs**
[LMBQ-373](https://lombiq.atlassian.net/browse/LMBQ-373)

[LMBQ-373]: https://lombiq.atlassian.net/browse/LMBQ-373?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Lombiq.Tests.UI/UITestBase.cs [hunk 1]
```diff
@@ -45,7 +45,7 @@ public abstract class UITestBase
 
             if (timeoutTask.IsCompleted)
             {
-                throw new TimeoutException($"The time allotted for the test ({timeout}) was exceeded.");
+                throw new TimeoutException($"The time allowed for the test ({timeout}) was exceeded.");
```
Comment (sarahelsaig) [inline Lombiq.Tests.UI/UITestBase.cs]: Revert this change. The phrase "the time allotted" is grammatically correct and appropriate for this specific context.
image

File: Lombiq.Tests.UI/Extensions/ScrollingUITestContextExtensions.cs [hunk 2]
```diff
@@ -27,8 +27,21 @@ public static void ScrollTo(this UITestContext context, By by)
     /// <summary>
     /// Scrolls to a particular set of coordinates in the document.
     /// </summary>
-    public static void ScrollTo(this UITestContext context, int x, int y) =>
+    public static void ScrollTo(this UITestContext context, int x, int y)
+    {
         context.ExecuteScript("window.scrollTo(arguments[0], arguments[1], \"instant\");", x, y);
```
Comment (sarahelsaig) [inline Lombiq.Tests.UI/Extensions/ScrollingUITestContextExtensions.cs]: Since you are using apostrophes below as well:

```suggestion
        context.ExecuteScript("window.scrollTo(arguments[0], arguments[1], 'instant');", x, y);
```

File: Lombiq.Tests.UI/Extensions/ScrollingUITestContextExtensions.cs [hunk 3]
```diff
@@ -27,8 +27,21 @@ public static void ScrollTo(this UITestContext context, By by)
...
+        context.TriggerHtmlReload();
+    }
+
+    /// <summary>
+    /// Sometimes after scrolling elements can't render properly, so we need to set the scale, to trigger the HTML
+    /// elements.
```
Comment (sarahelsaig) [inline Lombiq.Tests.UI/Extensions/ScrollingUITestContextExtensions.cs]:
```suggestion
    /// Sometimes elements don't render properly after scrolling. So we have to set the scale to re-render them.
```

**Review by sarahelsaig — CHANGES_REQUESTED** [review]

File: Lombiq.Tests.UI/UITestBase.cs (hunk 1 above)
Comment (DemeSzabolcs) [reply Lombiq.Tests.UI/UITestBase.cs]: Oh sorry 😅
